docs/modeling/models.rst

```diff
@@ -11,27 +25,24 @@ Fittable models can be linear or nonlinear in a regression analysis sense.
 
 In general models are instantiated by providing the parameter values that
 define that instance of the model to the constructor, as demonstrated in
-the section on :ref:`modeling-parameters`.
-
-Additionally, a `~astropy.modeling.Model` instance may represent a single model
-with one set of parameters, or a model *set* consisting of a set of parameters
-each representing a different parameterization of the same parametric model.
-For example one may instantiate a single Gaussian model with one mean, standard
-deviation, and amplitude.  Or one may create a set of N Gaussians, each one of
-which would be fitted to, for example, a different plane in an image cube.
-
-Regardless of whether using a single model, or a model set, parameter values
-may be scalar values, or arrays of any size and shape, so long as they are
-compatible according to the standard `Numpy broadcasting rules`_.  For example,
-a model may be instantiated with all scalar parameters::
-
-    >>> from astropy.modeling.models import Gaussian1D
-    >>> g = Gaussian1D(amplitude=1, mean=0, stddev=1)
-    >>> g
-    <Gaussian1D(amplitude=1.0, mean=0.0, stddev=1.0)>
+the section on :ref:`modeling-parameters`. A `~astropy.modeling.Model`
+instance may represent a single model with one set of parameters, or a model
+*set* consisting of a set of parameters each representing a different
+parameterization of the same parametric model. Regardless of whether using a
+single model, or a model set, parameter values may be scalar values, or arrays
+of any size and shape, so long as they are compatible according to the standard
+`Numpy broadcasting rules`_.
 
-Or it may use all array parameters.  For example if all parameters are 2x2
-arrays the model is computed element-wise using all elements in the arrays::
+.. _modeling-model-sets:
+
+Model Sets
+==========
+
+There are two ways, when instantiating a `~astropy.modeling.Model` instance, to
+create a model set. The first is to create the model using all array
+parameters, which follows Numpy broadcasting rules. For example, if all
+parameters are 2x2 arrays the model is computed element-wise using all elements
+in the arrays::
```

Comment:
This isn't quite right I'm afraid.  When you use arrays as parameters without specifying `n_models > 1` it does *not* imply a model set.  All it means is that the parameters have vector values, and it carries with it subtly different semantics from *model sets*.